manipvortex/manipulator.go

```diff
@@ -597,6 +598,10 @@ func (m *vortexManipulator) backgroundSync(ctx context.Context) {
 	for {
 		select {
 		case t := <-m.transactionQueue:
+			// Rate limit.
+			if err := limiter.Wait(ctx); err != nil {
+				zap.L().Warn("unable to rate limit", zap.Error(err))
+			}
```

Comment:
I am not sure we want a rate limiter for everything here. During normal operation there is no reason to have one. The rate limiter is needed after an outage. Otherwise we will normal traffic without a reason. In reality, if we want a rate limiter that makes sense it has to be a function of the delay of the backend. (ie if the backend starts slowing down and RTT times increase, then the rate limiter should throttle back. If the RTT times are very low, there is no reason to rate limit. So, I am not sure that this is the right approach here.